src/main/resources/feedbackQuestionMsqEditFormTemplate.html

```diff
@@ -64,7 +64,7 @@
       <input type="number" disabled class="form-control"
           name="${Const.ParamsNames.FEEDBACK_QUESTION_MSQ_MAX_SELECTABLE_CHOICES}"
           id="${Const.ParamsNames.FEEDBACK_QUESTION_MSQ_MAX_SELECTABLE_CHOICES}-${questionNumber}"
-          min="2" value="${msqMaxSelectableChoices}">
+          min="2" value="${msqMaxSelectableChoices}" required>
```

Comment:
When you add required here directly, when answering other type of questions like TEXT in Google Chrome browser, it will fail.

I will create an issue that describes possible workarounds you can take. Ultimately a redesign is necessary where we do not add all different type of questions in the same form

Reply:
Take a look at this stackoverflow post to know more

https://stackoverflow.com/questions/7168645/invalid-form-control-only-in-google-chrome

Reply:
@Crphang Ah! I was not aware of that! Thank you for notifying and providing the link.
But I have a confusion though. And that is, when we are in different question type, the max/min selectable fields are by default disabled right? So, in that case, shouldn't this solution work for our case? 

[This](https://stackoverflow.com/questions/22148080/an-invalid-form-control-with-name-is-not-focusable/22753533#22753533) answer used a similar technique to overcome this problem.

Reply:
Yes. But what we want to do is not to enable the disabled field, but to remove the `required` if it is not displayed. Look at this [PR](https://github.com/TEAMMATES/teammates/pull/8373#discussion_r175261726) I added required using javascript